Weather_App/Weather_App/Scenes/Map/MapViewModel.swift

```diff
@@ -119,11 +122,22 @@ extension MapViewModel: ViewModelType {
                     .map { statusFavorite }
             }
         
+        input.getSearchTextTrigger
+            .flatMapLatest({ searchText in
```

Comment:
```suggestion
            .flatMapLatest { searchText in
```

dùng trailing closure luôn nhé